content/composer/available-actions.md

```diff
@@ -6,42 +6,42 @@ draft: false
 weight: 300
 
 ---
+## Actions Bar
+On the top right you can find additional actions to perform on the current used blueprint.
 
-On the top right you can find additional actions to perform on the current opened blueprint.
-
-# Save
+### Save Blueprint
```

Comment:
yes - an invalid blueprint will be saved despite validation phase, the validation is done by the DSL parser (thirdparty) it's important to save it even if invalid for future editing. 
changed the title